lib/http.go

```diff
@@ -0,0 +1,102 @@
+package lib
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/qri-io/qri/config"
+)
+
+const jsonMimeType = "application/json"
+
+// HTTPClient implements the qri http client
+type HTTPClient struct {
+	Addr string
+}
+
+// NewHTTPClient instantiates a new HTTPClient
+func NewHTTPClient(addr string) *HTTPClient {
+	return &HTTPClient{
+		Addr: addr,
+	}
+}
+
+// Call resolves the action by mapping it to the correct API endpoint and passes on parameters and context info
+// currently this is a fairly flat process but should evolve in the future
```

Comment:
Comments like `currently this is...` get stale pretty quickly. I'd prefer we not add 'em